app/src/components/filterEntities/containers/filterEntitiesContainer.jsx

```diff
@@ -83,12 +83,19 @@ export class FilterEntitiesContainer extends Component {
     }
   };
 
-  handleAdd = (entity) => {
+  handleAdd = (entities) => {
     this.setState(
-      (prevState) => ({
-        values: { ...prevState.values, [entity.id]: entity.value },
-        visibleFilters: [...prevState.visibleFilters, entity.id],
-      }),
+      (prevState) => {
+        const entitiesObj = entities
```

Comment:
`const entitiesObj = entities.reduce((acc, entity) => ({...acc, [entity.id]: entity.value}), {});`

Reply:
Also, this change breaks adding filters by clicking a grid header (see `FilterEntitiesAddHandlerContainer`). I suggest supporting both cases (array and single entity)